javaslang/src/main/java/javaslang/collection/HashMultimap.java

```diff
@@ -25,10 +24,22 @@
  * @author Ruslan Sennov
  * @since 2.1.0
  */
-public final class HashMultimap<K, V> extends AbstractMultimap<K, V, HashMultimap<K, V>> implements Serializable {
+public final class HashMultimap<K, V> implements Multimap<K, V>, Serializable {
 
     private static final long serialVersionUID = 1L;
 
+    private final Map<K, Traversable<V>> back;
```

Comment:
Will rename it.

Note: Currently only LinkedHashMultimap and Multimaps is worth to take a look at. HashMultimap and TreeMultimap are far behind...